modules/extensions-support/src/main/java/org/mule/module/extension/internal/introspection/enricher/ContentTypeModelEnricher.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.module.extension.internal.introspection.enricher;
+
+import static org.mule.module.extension.internal.ExtensionProperties.ENCODING_PARAMETER_NAME;
+import static org.mule.module.extension.internal.ExtensionProperties.MIME_TYPE_PARAMETER_NAME;
+import org.mule.extension.annotation.api.ContentMetadataParameters;
+import org.mule.extension.api.introspection.DataType;
+import org.mule.extension.api.introspection.declaration.DescribingContext;
+import org.mule.extension.api.introspection.declaration.fluent.ParameterDeclaration;
+import org.mule.extension.api.runtime.ContentType;
+import org.mule.module.extension.internal.ExtensionProperties;
+import org.mule.module.extension.internal.model.AbstractAnnotatedModelEnricher;
+
+/**
+ * Enriches operations which were defined in methods annotated with {@link ContentMetadataParameters} so that
+ * parameters related to {@link ContentType} are added. In particular, it adds two parameters named
+ * {@link ExtensionProperties#MIME_TYPE_PARAMETER_NAME} and {@link ExtensionProperties#ENCODING_PARAMETER_NAME}.
+ * <p>
+ * Both attributes are optional, have no default value and accept expressions.
```

Comment:
what I want is to respect what's on the message currnetly, not the defaults (unless I'm not understanding what you meant)